src/client/components/localisation/model.ts

```diff
@@ -11,22 +11,22 @@ import { SkyboxModel } from './skybox/model'
 
 export class TimeModel {
   @observable time: number // seconds
-  @observable lastRenderTime: number // seconds
+  @observable lastPhysicsUpdate: number // seconds
```

Comment:
I have decoupled the physics loop from rendering loop — so this makes more sense.